modules/RTC/TraceablePeerConnection.js

```diff
@@ -307,6 +307,32 @@ const dumpSDP = function(description) {
     return `type: ${description.type}\r\n${description.sdp}`;
 };
 
+/**
+ * Determines whether the ICE candidate address is a relayed one.
+ *
+ * @param {RTCSessionDescription} rtcSessionDescription the session description
+ * that has the local ICE candidate address.
+ * @param {Array} addressArray an array in the form [ ip address, port ].
+ * @param {String} candidateType the candidate type: One of "host", "srflx",
+ * "prflx", or "relay"
+ * @return {boolean} true if the ICE candidate is a relayed one, otherwise
+ * false.
+ */
+function isRelay(rtcSessionDescription, addressArray, candidateType) {
```

Comment:
For me it seems that this method verifies if given address exists in given SDP. Why is it called isRelay and why does it take candidateType if it's supposed to check whether given ICE candidate is a relay one ? Am I missing something ?